ang/crmCaseType/timelineTable.html

```diff
@@ -61,6 +62,35 @@
         >
       </select>
     </td>
+    <td>
+      <select
+        ui-jq="select2"
+        ui-options="{dropdownAutoWidth : true}"
+        ng-model="activity.default_assignee_type"
+        ng-options="option.id as option.label for option in defaultAssigneeTypes"
+        ng-change="clearActivityDefaultAssigneeValues(activity)"
+      ></select>
+
+      <p ng-if="activity.default_assignee_type === defaultAssigneeTypesIndex.BY_RELATIONSHIP.id">
+        <select
+          ui-jq="select2"
+          ui-options="{dropdownAutoWidth : true}"
+          ng-model="activity.default_assignee_relationship"
+          ng-options="option as option.text for option in relationshipTypeOptions"
+          required
+        ></select>
+      </p>
+
+      <p ng-if="activity.default_assignee_type === defaultAssigneeTypesIndex.SPECIFIC_CONTACT.id">
+        <input
+          type="text"
+          ng-model="activity.default_assignee_contact"
+          placeholder="- select contact -"
```

Comment:
Not quite sure lower capitalised placeholder would look good. I think it should be `- Select contact -` but better to consult with PM.